src/main.cu

```diff
@@ -587,168 +628,186 @@ typedef struct Experiment {
 	    this->popSizeIncrement = popSizeIncrement;
 	    this->totalRuns = totalRuns;
 	    this->ensembleSizes = ensembleSizes;
+        this->repetitions = repetitions;
 	    this->steps = steps;
-	    this->spatial = spatial;	
+	    this->spatial = spatial;
 	}
 	std::string title;
 	unsigned int initialPopSize, finalPopSize, popSizeIncrement;
 	unsigned int totalRuns;
 	std::vector<unsigned int> ensembleSizes;
+    int repetitions;
 	unsigned int steps;
 	bool spatial;
 } Experiment;
 
 int main(int argc, const char ** argv) {
+    
+    const int repetitions = 3;
 
-    Experiment smallPopBruteForce("small_pop_brute_force", 128, 1024, 128, 60, std::vector<unsigned int> {1, 2, 4, 5, 6, 10, 12, 15, 20, 30, 60}, 500, false);
-    Experiment largePopBruteForce("large_pop_brute_force", 2048, 8192, 2048, 60, std::vector<unsigned int> {1, 2, 4, 5, 6, 10, 12, 15, 20, 30, 60}, 500, false);
-    //Experiment veryLargePopBruteForce("very_large_pop_brute_force", 5000, 5000, 5000, 60, std::vector<unsigned int> {1, 2, 4, 5, 6, 10, 12, 15, 20, 30, 60}, 500, false);
+    Experiment smallPopBruteForce("small_pop_brute_force", 128, 1024, 128, 60, std::vector<unsigned int> {1, 2, 4, 5, 6, 10, 12, 15, 20, 30, 60}, repetitions, 500, false);
+    Experiment largePopBruteForce("large_pop_brute_force", 2048, 8192, 2048, 60, std::vector<unsigned int> {1, 2, 4, 5, 6, 10, 12, 15, 20, 30, 60}, repetitions, 500, false);
+    Experiment veryLargePopBruteForce("very_large_pop_brute_force", 5000, 5000, 5000, 60, std::vector<unsigned int> {1, 2, 4, 5, 6, 10, 12, 15, 20, 30, 60}, repetitions, 500, false);
     
-    Experiment smallPop("small_pop", 128, 1024, 128, 60, std::vector<unsigned int> {1, 2, 4, 5, 6, 10, 12, 15, 20, 30, 60}, 500, true);
-    Experiment largePop("large_pop", 2048, 8192, 2048, 60, std::vector<unsigned int> {1, 2, 4, 5, 6, 10, 12, 15, 20, 30, 60}, 500, true);
-    //Experiment veryLargePop("very_large_pop_brute_force", 5000, 20000, 5000, 60, std::vector<unsigned int> {1, 2, 4, 5, 6, 10, 12, 15, 20, 30, 60}, 500, true);
+    Experiment smallPop("small_pop", 128, 1024, 128, 60, std::vector<unsigned int> {1, 2, 4, 5, 6, 10, 12, 15, 20, 30, 60}, repetitions, 500, true);
+    Experiment largePop("large_pop", 2048, 8192, 2048, 60, std::vector<unsigned int> {1, 2, 4, 5, 6, 10, 12, 15, 20, 30, 60}, repetitions, 500, true);
+    Experiment veryLargePop("very_large_pop_brute_force", 5000, 20000, 5000, 60, std::vector<unsigned int> {1, 2, 4, 5, 6, 10, 12, 15, 20, 30, 60}, repetitions, 500, true);
```

Comment:
This has been uncommented but is not in the vector of experiments? Doesn't need including, just making sure you didn't also mean to add it to the vector.

Reply:
I didn't mean to include it in the vector and saw no reason for it to stay commented out given the experiments run are controlled via the vector anyway